Added users table view with the requested characteristics

andisima opened the pull request:
# Description
What did you implemented/Why did you implemented this:
 - Explain the changes you’ve made. 
 - It doesn’t need to be fancy and you don’t have to get to technical, yet. 
 - Just explicit what you have implemented at a high level. 
 - Let the reviewer know the overall effect of the PR.
 - Reference a ticket in your issue tracker. Explain what the change is and then and only then reference the ticket.
 - The “why” is a chance to explain both the engineering goal, but also a some business objective that is satisfied or moved along.
 
 Examples:
 - What?: I've added support for authentication to implement Key Result 2 of OKR1. It includes 
model, table, controller and test. For more background, see ticket #JIRA-123.
 - Why?: These changes complete the user login and account creation experience. See #JIRA-123 for more information.
 
 # Testing
 Help me how can I test or look at the changes
 
 Example:
  - I've added coverage for testing all new methods. I used Faker for a few random user emails and names.
  
  ## Screenshots 
If applicable, include screenshots of the results or screenshots that help to see changes

Commit:
Added users table view with the requested characteristics

CarlosSe reviewed (commented):
Looks good, just that are two types of a button for deleting a user, one in the top button and another in-line with the record, I think that just one method is easy to use and less prone to error and misinterpretation, what do you think?

analauraarreola committed:
Deleting an extra button
I deleted an exra button of this issue because it was reppeated

Now the issue is ready
:qw

analauraarreola commented:
I did anither commit to this pull request
